Ports/vim/Vim.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+PORT_DIR=Vim
+INSTALLOPTS="DESTDIR=$SERENITY_ROOT/Root/"
+
+function fetch() {
+    run_fetch_git "https://github.com/vim/vim.git"
+}
+
+function configure() {
+    (
```

Comment:
Ok, I'd suggest we add a new "helper" to .port_include.sh:

run_append_file

Just to make the *intent* clear, and also make it easier to lint/tool, or port the ports to another language than shell script later on if need be. Basically, I want to try to pretend that these aren't shell scripts as much as possible (outside .port_include).

I want to revisit my own perl invocations in the bash port too, and replace them in a similar way: (run_replace helper function rather than directly invoking perl).